NeuralAudio/NeuralModel.cpp

```diff
@@ -83,16 +83,21 @@ namespace NeuralAudio
 
 	NeuralModel* NeuralModel::CreateFromFile(std::filesystem::path modelPath)
 	{
-		EnsureModelDefsAreLoaded();
-
 		std::ifstream jsonStream(modelPath, std::ifstream::binary);
 
+		return CreateFromStream(jsonStream, modelPath.extension());
+	}
+
+	NeuralModel* NeuralModel::CreateFromStream(std::basic_istream<char>& jsonStream, std::string extension)
```

Comment:
modelPath.extension() is still a std::filesystem::path. Looks like gcc is ok with auto-converting it to std::string, but the windows compiler doesn't like it. Doing "modelPath.exenstion().string()" would fix it, but I think I'd prefer to have the "CreateFromStream" take std::filesystem::path for the extension.

Reply:
sounds reasonable to me, should I push that change or you want to do it yourself?

Reply:
fixed just now